packages/core/addon/chart-builders/date-time.js

```diff
@@ -253,7 +253,7 @@ export default Ember.Object.extend({
       rowData: Ember.computed('x', 'tooltipData', function() {
         return get(this, 'tooltipData').map(series => {
           // Get the full data for this combination of x + series
-          let dataForSeries = byXSeries.getDataForKey(get(this, 'x') + series.name) || [];
+          let dataForSeries = get(builder, 'byXSeries').getDataForKey(get(this, 'x') + series.name) || [];
```

Comment:
why is this change necessary?

Reply:
Because closing over the data as a pojo in the tooltip builder, means that the tooltip won't see updates when `byXSeries` in the object changes. Therefore, if `byXSeries` changes the tooltip can't access the row data correctly.  By closing over the builder instead it can always extract up-to-date `byXSeries`